source/agora/crypto/Schnorr.d

```diff
@@ -130,6 +130,26 @@ public struct Sig
 
     static assert(Signature.sizeof == typeof(this).sizeof);
 
+    /// construct from Point and Scalar
+    public this (in Point R, in Scalar s) pure nothrow @nogc @safe
+    {
+        this.R = R;
+        this.s = s;
+    }
+
+    unittest
+    {
+        import std.format;
+
+        auto R = Point("0x921405afbfa97813293770efd55865c01055f39ad2a70f2b7a04ac043766a693");
+        assert(R.isValid()); // valid Point
+        auto s = Scalar("0x074360d5eab8e888df07d862c4fc845ebd10b6a6c530919d66221219bba50216");
+        assert(s.isValid()); // valid Scalar
+        auto sig = Sig(R, s);
```

Comment:
Testing actor seems a bit overkill but it makes for a good documentation I guess :P